src/tests/helpers.js

```diff
@@ -463,6 +463,12 @@ export const mockCheckboxInput = {
     })
     .build();
  */
+
```

Comment:
The [internal pipelines are green](https://gitlab.com/remote-com/employ-starbase/dragon/-/merge_requests/20741). I reverted the package.json, ready for approval!

P.S. I won't do a new release after merge because this is an internal chore that doesn't affect consumers. In future release, this commit can even be deleted from the CHANGELOG.